### DISCUSSION TITLE
Update PhpMyAdmin and MySQL Failed

### DISCUSSION THREAD

**Discussion by tienloc1** [leokhoa/laragon#472] [Q&A]
Hello everyone,

I tried to update PhpMyAdmin and MySQL in Laragon 5 and it failed. I hope I can get some advice to make it work.

1. For PhpMyAdmin:
- Download it from: https://www.phpmyadmin.net/downloads/
- Add the new files in: laragon/etc/apps/phpMyAdmin
- Stop/Start Laragon
Error: sitename.test unable to access with all sites.

2. For MySQL:
- Download it from: https://dev.mysql.com/downloads/mysql/
- Add the new files in: laragon/bin/mysql
- Stop/Start Laragon
Error: mysqli_real_connect() mysql server has gone away/ while reading greeting packet. pid=16000.

Nothing work.

**Comment by tienloc1** [comment]
For PhpMyAdmin: after shutdown and working today, it's working now, not sure why, lol.

For MySQL: this is my second try, but it still doesn't work.

**Reply by matthiku** [reply]
I do have the same issue with MySQL.
What is the URL for your PhpMyAdmin page?

**Reply by tienloc1** [reply]
PhpMyAdmin URL: localhost/myphpadmin, I'm still stuck with the MySQL update.

**Reply by josealgardataborges** [reply]
MySQL: did you place the unzipped folders in the right folder inside C:\Laragon\bin\mysql\, like so?

MySQL5
![image](https://user-images.githubusercontent.com/8929655/231484324-c5c6e990-8f3d-4091-8895-2865a9e5362e.png)

MySQL8
![image](https://user-images.githubusercontent.com/8929655/231484490-880a55da-f92b-4951-8da0-cb86c0e891ef.png)

**Reply by josealgardataborges** [reply]
Follow closely this tutorial and you should be fine: https://medium.com/@leovn/how-to-add-another-mysql-version-to-laragon-23d5032b6d4e

**Reply by matthiku** [reply]
Yes, I did and followed exactly the steps as outlined on medium.com.
Do you know that the default root password is for  MySql after installation on the first run?
Or do you have to runsome database initialization process?

**Reply by tienloc1** [reply]
I'm finally getting it to work for me.

You can try to check it yourself @matthiku 

- Stop the laragon
- Add the new version in: laragon/bin/mysql
- Go to: laragon/data/ and in here, you will see mysql and mysql-8
+ Backup the mysql (the real data of mine)
+ Delete mysql-8
+ Create a new mysql-8 by copy mysql and renaming it
- Start the laragon
- Bump, it's working for me.

**Reply by yosbp** [reply]
> I'm finally getting it to work for me.
> 
> You can try to check it yourself @matthiku
> 
> * Stop the laragon
> * Add the new version in: laragon/bin/mysql
> * Go to: laragon/data/ and in here, you will see mysql and mysql-8
> 
> * Backup the mysql (the real data of mine)
> * Delete mysql-8
> * Create a new mysql-8 by copy mysql and renaming it
> 
> * Start the laragon
> * Bump, it's working for me.

Works for me. Ty!
